admin/package.json

```diff
@@ -63,6 +63,7 @@
     "tedious-connection-pool": "^1.0.5",
     "uuid": "^3.1.0",
     "validator": "^9.2.0",
+    "winston": "^2.4.0",
```

Comment:
this will likely clash with my addition when it hits master.  shouldn't be an issue